tests/src/rules/group-exports.js

```diff
@@ -0,0 +1,70 @@
+import { test } from '../utils'
+import { RuleTester } from 'eslint'
+import rule from 'rules/group-exports'
+
+const ruleTester = new RuleTester()
+
+ruleTester.run('group-exports', rule, {
+  valid: [
+    test({ code: 'export const test = true' }),
+    test({ code: 'export default {}\nexport const test = true' }),
+    test({ code: [
+      'const first = true',
+      'const second = true',
+      'export { first,\nsecond }',
+    ].join('\n') }),
+    test({ code: 'export default {}\n/* test */\nexport const test = true'}),
+    test({ code: 'export default {}\n// test\nexport const test = true'}),
+    test({ code: 'export const test = true\n/* test */\nexport default {}'}),
+    test({ code: 'export const test = true\n// test\nexport default {}'}),
+    test({ code: 'module.exports = {} '}),
+    test({ code: 'module.exports = { test: true,\nanother: false }' }),
+    test({ code: 'exports.test = true' }),
+  ],
+  invalid: [
+    test({
+      code: [
+        'export const test = true',
+        'export const another = true',
+      ].join('\n'),
+      errors: [
+        'Multiple named export declarations',
+        'Multiple named export declarations',
+      ],
+    }),
+    test({
+      code: [
+        'module.exports = {}',
+        'module.exports.test = true',
+        'module.exports.another = true',
+      ].join('\n'),
+      errors: [
+        'Multiple CommonJS exports',
+        'Multiple CommonJS exports',
+        'Multiple CommonJS exports',
```

Comment:
Assignment to the `exports` object is covered [here](https://github.com/benmosher/eslint-plugin-import/pull/721/files#diff-7c6599b72765e69ef4f27ae9f27ff175R91).